go/vt/vtgate/vtgate.go

```diff
@@ -291,6 +293,27 @@ func Init(
 	tabletTypesToWait []topodatapb.TabletType,
 	pv plancontext.PlannerVersion,
 ) *VTGate {
+	ts, err := serv.GetTopoServer()
+	if err != nil {
+		log.Fatalf("Unable to get Topo server: %v", err)
```

Comment:
This is fine for now. It follows the pattern that is being used for other fatal errors. We should go back and review whether any of these should cause vtgate to exit using `log.Exitf`.